src/Ordering.Domain/AggregatesModel/OrderAggregate/Order.cs

```diff
@@ -195,4 +195,15 @@ public decimal GetTotal()
     {
         return _orderItems.Sum(o => o.GetUnits() * o.GetUnitPrice());
     }
+    
+    
+    public DateTime GetOrderDate()
```

Comment:
I'd simply have an OrderDate property (and a BuyerId property). The general feeling in the team is that the patterns used here - e.g. avoiding properties and having private fields with Set/Get methods - is an anti-pattern.

But this can be done in a separate PR. Otherwise this can be an expression-bodied method for nicer C#.

Reply:
Great to hear that, I was also not a fan of using methods for properties. However, I was perhaps overly cautious about rewriting everything since I wasn't sure if there was another reason behind this approach.

Reply:
I'm honestly not sure where this comes from - there's the comment attributing this to "DDD best practices", but I'm not sure exactly what that refers to. In any case, it's fine to leave things as-is and clean this up in a separate pass later.